guides/common/modules/proc_changing-the-environment-of-a-host.adoc

```diff
@@ -6,7 +6,7 @@ Use this procedure to change the environment of a host.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
-. Click the vertical ellipsis in the *Content view details* card and select *Edit content view assignment*.
+. Click the options icon on the *Content view environment* card and select *Edit content view environments*.
```

Comment:
```suggestion
. On the *Content view environment* card, click the options icon and select *Edit content view environments*.
```
This will guide the user better because they have to look for the card first before they look for the icon.